### PR TITLE
simplify contacts form guid extraction

### PR DESCRIPTION
**Pull request by sloev**
# Description of change
(write a short description or paste a link to JIRA)

# Manual QA steps
 - 
 
# Risks
 - 
 
# Rollback steps
 - revert this branch

### CODE REVIEW

**Review by paddie — APPROVED** [review]

File: tap_hubspot/hubspot.py [hunk 1]
```diff
@@ -315,10 +300,19 @@ def get_submissions(self):
...
+        def merge_guids():
+            for guid in guids_from_contacts:
+                guids_from_endpoint.discard(
+                    guid
+                )  # does not raise keyerror if not exists
+                yield guid
+            for guid in guids_from_endpoint:
+                yield guid
+
+        for guid in merge_guids():
```
[the remark this answers is not in the document]
Comment (sloev) [reply tap_hubspot/hubspot.py:315]: @paddie no cuz they get removed here: https://github.com/dreamdata-io/tap-hubspot/pull/26

[the remark this answers is not in the document]
Comment (sloev) [reply tap_hubspot/hubspot.py:315]:
```
           for guid in guids_from_contacts:
                guids_from_endpoint.discard(
                    guid
                )  # does not raise keyerror if not exists
                yield guid
            for guid in guids_from_endpoint:
                yield guid

```

[the remark this answers is not in the document]
Comment (sloev) [reply tap_hubspot/hubspot.py:315]: guids_from_endpoint.discard(
                    guid
                )
                makes sure to discard already examined guids from the contacts

[the remark this answers is not in the document]
Comment (paddie) [reply tap_hubspot/hubspot.py:315]: nice 👍

**Pull request merged**